### PR TITLE
Add support for === and !==

### PR DESCRIPTION
**Pull request by dawehner**
Solves #2358

### CODE REVIEW

File: lib/Twig/Node/Expression/Binary/StrictEqual.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+class Twig_Node_Expression_Binary_StrictEqual extends Twig_Node_Expression_Binary
+{
+    public function operator(Twig_Compiler $compiler)
+    {
+        return $compiler->raw('===');
```
Comment (SpacePossum) [inline lib/Twig/Node/Expression/Binary/StrictEqual.php]: I'm confused, this is causing an exception being thrown with a message informing that `same as` should be used right? (https://github.com/twigphp/Twig/issues/2358#issuecomment-271995283)

Comment (dawehner) [inline lib/Twig/Node/Expression/Binary/StrictEqual.php]: Oh you are totally right. I was just not really awake this morning.

**Comment by robfrawley** [comment]
@fabpot I absolutely understand why you chose to support `same as` - to allow easy adoption by those unfamiliar with PHP. That said, I can't understand the number of times I've typed in `===` only to compile the template and go "oh, right, I can't use that" - it would be nice to document the preferred syntax of `same as` while *also* allow the use of `===` for those more accustomed to it.

**Comment by stof** [comment]
What @fabpot accepted in the issue discussion is *not* adding support for it, but throwing an helpful exception.

**Comment by fabpot** [comment]
Closing this one as this is not what needs to be done.

**Comment by robfrawley** [comment]
Disappointed, but fair enough @fabpot. I understand why you don't want to pollute the language with multiple syntaxes.

**Pull request closed without merging**